### PR TITLE
Update EMIP00634

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
According to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2565.

### CODE REVIEW

File: EMIP/1-1000/EMIP00634.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                   <msItem xml:id="ms_i1">
                     <locus from="1r" to="15r" facs="007"/>
-                  <title type="complete" ref="LIT5944CommMystatgogia"/>
+                  <title type="complete" ref="LIT5944Mystagogia"/>
```
Comment (eu-genia) [inline EMIP/1-1000/EMIP00634.xml:46]: ok but you could have also simply call your new file LIT5944CommMystatgogia and not change anything here

Comment (CarstenHoffmannMarburg) [reply EMIP/1-1000/EMIP00634.xml:46]: Unfortunately, I have already given a new name. I will do it the easier for the next time.

Comment (eu-genia) [reply EMIP/1-1000/EMIP00634.xml:46]: I repeat. You can rename the file. Then this PR can be closed without merging.

Comment (CarstenHoffmannMarburg) [reply EMIP/1-1000/EMIP00634.xml:46]: I renamed the file and closed and deleted this branch.

**Pull request closed without merging**